docs/debugging_tips.md

```diff
@@ -35,19 +35,135 @@ configuration, diagnostic, and troubleshooting data to be provided to Red Hat Te
+                 firewall_tables, hardware, host, i18n, insights, kdump,
+                 kernel, krb5, kvm, ldap, libraries, libvirt, login,
+                 logrotate, logs, memory, multipath, networking,
+                 networkmanager, openhpi, pam, pci, process, processor,
+                 python, release, rpm, selinux, services, ssh,
+                 subscription_manager, sudo, system, systemd, sysvipc
+                 tuned, udev, udisks, unbound, vhostmd, yum
+ virt            kvm, libvirt, vhostmd, xen
+ webserver       perl
+
+ 24 profiles, 91 plugins
 ```
 
-The report archives can be found in the `/var/tmp/sosreport-*` files.
+Log into the host running MicroShift and execute the following command to generate an obfuscated
+report that should not contain sensitive information (in this example only microshift plugin was
+enabled. Your system's output may vary):
+```bash
+$ sos report --batch --clean
+
+sosreport (version 4.3)
+
+This command will collect diagnostic and configuration information from
+this Red Hat Enterprise Linux system and installed applications.
+
+An archive containing the collected information will be generated in
+/var/tmp/sos.cn3_s5h3 and may be provided to a Red Hat support
+representative.
+
+Any information provided to Red Hat will be treated in accordance with
+the published support policies at:
+
+        Distribution Website : https://www.redhat.com/
+        Commercial Support   : https://www.access.redhat.com/
+
+The generated archive may contain data considered sensitive and its
+content should be reviewed by the originating organization before being
+passed to any third party.
+
+No changes will be made to system configuration.
+
+
+ Setting up archive ...
+ Setting up plugins ...
+ Running plugins. Please wait ...
+
+  Starting 1/1   microshift      [Running: microshift]
+
+  Finished running plugins
+
+Found 1 total reports to obfuscate, processing up to 4 concurrently
+
+sosreport-microshift-2-2023-03-23-ylwbkjc :        Beginning obfuscation...
+sosreport-microshift-2-2023-03-23-ylwbkjc :        Obfuscation completed
+
+Successfully obfuscated 1 report(s)
+
+Creating compressed archive...
+
+A mapping of obfuscated elements is available at
+	/var/tmp/sosreport-microshift-2-2023-03-23-ylwbkjc-private_map
+
+Your sosreport has been generated and saved in:
+	/var/tmp/sosreport-microshift-2-2023-03-23-ylwbkjc-obfuscated.tar.xz
+
+ Size	4.48KiB
+ Owner	root
+ sha256	282c06266297396aeeb1a21579811ac0f2db43d27bee9cf1cd55b997c952cb24
+
+Please send this file to your support representative.
+```
+> Sos must always run with root privileges.
 
+The output file was generated in `/var/tmp/` dir, and a checksum file is also generated.
 ```bash
-$ sudo ls -tr /var/tmp/sosreport-* | tail -2
-/var/tmp/sosreport-host0-2022-11-24-pvbcaji-obfuscated.tar.xz
-/var/tmp/sosreport-host0-2022-11-24-pvbcaji-obfuscated.tar.xz.sha256
+$ sudo ls -tr /var/tmp/
```

Comment:
Do we want to leave the original command to only display the last two archives? The current command will display everything in the /var/tmp

Reply:
Fair enough, it should be mostly empty though.